lib/pbench/server/api/resources/__init__.py

```diff
@@ -1142,9 +1175,7 @@ def _check_authorization(
             Any authenticated ADMIN user can update/delete any data.
 
         Args:
-            user_id: The client's requested user ID (as a string), or None
-            access: The client's requested access parameter, or None
-            check_role: Override self.role for access check
+            ApiAuthorization object with type, role, and user information
 
         Raises:
             UnauthorizedAccess: The user isn't authorized for the requested
```

Comment:
Now also raises `UnauthorizedAdminAccess`.

Reply:
And that would indeed be useful to point out, although technically it's correct now as `UnauthorizedAdminAccess` **isa** `UnauthorizedAccess`. 😏

Reply:
Um...true...you've got me, there.  😀